application/models.py

```diff
@@ -0,0 +1,10 @@
+from application import db
+
+
+class Task(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    task_descr = db.Column(db.String(300))
+    status = db.Column(db.String(30), index=True)
```

Comment:
what data will be stored in _status_ field of your _Task_ model? What are you going to have?
if it is like: **Done**, **Not Done** then mb you can have this field as **Boolean**?
or you can use this one:
[https://stackoverflow.com/questions/63034703/flask-and-sqlalchemys-column-type](url)
time to learn what is **Enum** in Python

Reply:
done. Enum added